### PR TITLE
Feature/cv Martí Malek

### PR DESCRIPTION
**Pull request by martimalek**
I've used the `<hr>` tag because it is defined as semantic.

[https://www.w3schools.com/tags/tag_hr.asp](url)

### CODE REVIEW

File: staff/marti-malek/cv/index.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CV</title>
+</head>
+
+<body>
+    <header>
+        <h1>Martí Malek</h1>
+        <img src="superrandom" alt="My Image">
+        <h3>Personal Information</h3>
+        <address>
```
Comment (lexbarba) [inline staff/marti-malek/cv/index.html]: Coool!

File: staff/marti-malek/cv/index.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        Address: C/Pedrell N.80 <b>Barcelona</b><br>
+        Tlf: 697 501 950<br>
+        e-mail: <a href="mailto:martimaledot@hotmail.es">martimalekdot@hotmail.es</a>
+        </address>
+        <h3>Profile</h3>
+        <p><i>Very hard-working person with energy and skills</i></p>
```
Comment (lexbarba) [inline staff/marti-malek/cv/index.html]: This is to style it!

Use the <i> element only when there is not a more appropriate semantic element, such as:

<em> (emphasized text)
<strong> (important text)
<mark> (marked/highlighted text)
<cite> (the title of a work)
<dfn> (a definition term)

[https://www.w3schools.com/tags/tag_i.asp](url)

File: staff/marti-malek/cv/index.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+            <li>18 years of expertise as a day-dreamer</li>
+            <li>More than 10 years of expertise as a person</li>
+        </ul>
+    </section>
+    <section>
+        <h2>Education</h2>
+        <ul>
+            <li>High school</li>
+            <li>Currently doing a Full-Stack Web Development Bootcamp in <a href="https://www.skylabcoders.com/es"><b>SkylabCodersAcademy</b></a></li>
+            <li>Currently studying chinese in <a href="https://www.eoibd.cat/es/"><b>Escola Oficial d'Idiomes</b></a></li>
```
Comment (lexbarba) [inline staff/marti-malek/cv/index.html]: This is to style it! [https://www.w3schools.com/tags/tag_b.asp](url)

File: staff/marti-malek/cv/index.html [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+</head>
+
+<body>
+    <header>
+        <h1>Martí Malek</h1>
+        <img src="superrandom" alt="My Image">
+        <h3>Personal Information</h3>
+        <address>
+        Address: C/Pedrell N.80 <b>Barcelona</b><br>
+        Tlf: 697 501 950<br>
```
Comment (quinwacca) [inline staff/marti-malek/cv/index.html]: You can add hyperlink to phone number too, similar way to hyperlink the email adress:
```javascript
<a href="tel:8648726348">8648726348</a>
```

File: staff/marti-malek/cv/index.html (hunk 1 above)
Comment (martimalek) [reply staff/marti-malek/cv/index.html]: Thx!

File: staff/marti-malek/cv/index.html (hunk 4 above)
Comment (martimalek) [reply staff/marti-malek/cv/index.html]: Thx, did it in the second commit!

Comment (martimalek) [reply staff/marti-malek/cv/index.html]: Thx! I used it on the second commit!

File: staff/marti-malek/cv/index.html (hunk 3 above)
Comment (martimalek) [reply staff/marti-malek/cv/index.html]: Fixed it! ;)

File: staff/marti-malek/cv/index.html (hunk 2 above)
Comment (martimalek) [reply staff/marti-malek/cv/index.html]: Fixed it! 👍

**Pull request merged**